src/main.ts

```diff
@@ -1,8 +1,11 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { ValidationPipe } from '@nestjs/common';
+import * as process from 'process';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
-  await app.listen(3000);
+  app.useGlobalPipes(new ValidationPipe());
```

Comment:
Do changes here please find below link


https://stackoverflow.com/questions/55414165/why-does-whitelist-dont-get-error-with-wrong-model-nestjs#:~:text=The%20option%20whitelist%20only%20strips%20unknown%20values%20when%20the%20json%20is%20transformed%20into%20your%20dto%20class%20without%20throwing%20an%20error%2C%20so%20in%20your%20case%20the%20property%20test%20is%20silently%20removed.